.gitignore

```diff
@@ -6,4 +6,7 @@ _build/
 dist/
 MANIFEST
 Axelrod.egg-info
-.hypothesis/eval_source/
+.hypothesis/
```

Comment:
I think there might be a better way of making the db persist on travis (David mentioned something about this on the hypothesis irc channel). Something about a cached folder... If the db is causing problems we can leave this as it is and open and I can look in to that...